portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -721,6 +721,18 @@ definition {
 		}
 	}
 
+	macro viewFieldReferenceFromGridField {
+		Click(
+			key_fieldName = "Grid",
+			locator1 = "Form#FIELD_BODY");
+
+		Form.gotoAdvancedTab();
+
+		AssertTextEquals(
+			locator1 = "Form#FIELD_REFERENCE",
+			value1 = "${key_fieldReference}");
+	}
```

Comment:
Hi @ewertonluna , instead of creating a macro that asserts the Field Reference only for the Grid Field. It would be better if this macro works for any type of field. 
I imagine that if you add another parameter key_fieldName, instead of hardcoding "Grid" it will work. Thanks in advance.

Reply:
Hi @mlan0! Thank you for the review! I just made the changes and resend it. Thanks.